src/main/java/com/example/config/SimpleConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.example.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import com.example.filter.SimpleInterceptor;
+
+@Configuration
+public class SimpleConfig implements WebMvcConfigurer{
+	private SimpleInterceptor interceptor;
+	
+	@Autowired
+	public SimpleConfig(SimpleInterceptor interceptor) {
+		this.interceptor = interceptor;
+	}
+	
+	@Override
+    public void addInterceptors(InterceptorRegistry registry) {
```

Comment:
Always use either tabs or 4 spaces for indentation, but do not mix both. Most of the projects I've encountered use 4 spaces, so that is preferable unless the project specifically indents with tabs only.